tests/http/entity-caching.yml

```diff
@@ -0,0 +1,41 @@
+config: !file tests/http/config/entity-caching.graphql
+name: Caching
+mock:
+  - request:
+      url: http://example.com/bars
+    response:
+      body:
+        - id: 1
+          foo:
+            id: 2
+        - id: 3
+          foo:
+            id: 4
+        - id: 5
+          foo:
+            id: 6
+        - id: 7
+          foo:
+            id: 8
+assert:
```

Comment:
is this test somehow tests the bug?

Reply:
not really, this is to test that the new changes don't mess up anything and to increase code coverage. I have manually tested it and I am not sure if it's possible to write an integration test like this to check that